kairon/shared/data/data_validation.py

```
@@ -0,0 +1,181 @@
+    @staticmethod
+    def validate_llm_prompts(llm_prompts: list):
+        error_list = []
+        system_prompt_count = 0
+        history_prompt_count = 0
+        for prompt in llm_prompts:
+            if prompt.get('hyperparameters') is not None:
+                hyperparameters = prompt.get('hyperparameters')
+                for key, value in hyperparameters.items():
+                    if key == 'similarity_threshold':
+                        if not (0.3 <= value <= 1.0) or not (
+                                isinstance(value, float) or isinstance(value, int)):
```

Ruff

Use a single `if` statement instead of nested `if` statements

(SIM102)